layouts/funding_calls/list.html

```diff
@@ -40,154 +37,80 @@ <h4>Potential funding sources</h4>
-                            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
-                          </div>
-                          <div class="modal-body">
-                            <div class="row mt-1">
-                              <div class="col">
-                                <b>Application deadline:</b> ${item.deadline_time ? long_date + " - " + item.deadline_time : long_date}
-                              </div>
-                            </div>`;
-
-                            if(funder){ 
-                            blockHTML += `
-                            <div class="row mt-2">
-                              <div class="col">
-                                <b>Funder:</b> ${ item.funder }
-                              </div>
-                            </div>`;
-                            } 
-
-                            if(item.description) {
-                              blockHTML += `
-                              <div class="row mt-2">
-                                <div class="col"><b>Brief description:</b><br>
-                                  ${item.description}
-                                </div>
-                              </div>`;
-                            }
-
-                            if(item.applicant){
-                              blockHTML += `
-                              <div class="row mt-2">
-                                <div class="col"><b>Who can apply?</b><br>
-                                  ${item.applicant}
-                                </div>
-                              </div>`;
-                            }
-                            
-
-                            blockHTML += `<div class="row mt-3">
-                              <p>For more information, visit the link below.</p>
-                              <div class="col-md">
-                                <a target="_blank" href="${item.url}"><i class="bi bi-globe"></i> Call webpage</a>
-                              </div>
-                            </div>
-                          </div>
-                          <div class="modal-footer">
-                            <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
-                          </div>
-                        </div>
-                      </div>
-                    </div>
-                  </div>
-                </div>`;
-
-            blockHTML += `
+        const closeDate = new Date(item.closes);
+        const closeDateFormatted = closeDate.toLocaleString('en-SE', dateFormatSettings);
+        blockHTML += `
+        <div class="col-md-6 col-lg-4">
+          <div class="card-box">
+            <a href="${item.href}" target="_blank">
+              <h5>${item.title}</h5>
+              <div>
+                <b>Deadline:</b> ${closeDateFormatted + ' (Swedish time)'}
```

Comment:
Not sure but minor suggestion, direct interpolation can avoid an extra concatenation if we can use it here like this.

```suggestion
                <b>Deadline:</b> ${closeDateFormatted} (Swedish time)
```

Reply:
My bad, I forgot it was already a string interpolation. Good catch 😄